pyfar/plot/line.py

```diff
@@ -257,7 +257,7 @@ def group_delay(signal, unit=None, xscale='log', ax=None, style='light',
 def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
                 yscale='linear', unit=None, window='hann', window_length=1024,
                 window_overlap_fct=0.5, cmap=mpl.cm.get_cmap(name='magma'),
-                ax=None, style='light'):
+                colorbar=True, ax=None, style='light'):
```

Comment:
Colorbar parameter is missing in the parameter list below

Reply:
Thanks. I added that.